src/clusterfuzz/_internal/bot/tasks/trials.py

```diff
@@ -61,17 +62,24 @@ def __init__(self):
       for config in trials_config:
         if config['app_name'] != app_name:
           continue
-        self.trials[config['app_args']] = config['probability']
+        self.trials[config['app_args']] = {'probability': config['probability']}
+        if 'contradicts' in config:
+          self.trials[config['app_args']]['contradicts'] = config['contradicts']
     except Exception as e:
       logs.log_warn('Unable to parse config file: %s' % str(e))
       return
 
   def setup_additional_args_for_app(self):
     """Select additional args for the specified app at random."""
-    trial_args = [
-        app_args for app_args, probability in self.trials.items()
-        if random.random() < probability
-    ]
+    trial_args = []
+    contradicts = []
+    for app_args, flag_data in self.trials.items():
+      if random.random(
```

Comment:
Can you add reformat this so this is ealier to read by adding ( ) surrounding the entire if condition.

i.e. something like

```python
if (random.random() < flag_data[...] and
    app_args not in contradicts):
  ...

```